js/data/bucket.js

```diff
@@ -58,8 +58,6 @@ class Bucket {
                 options.featureIndex.insert(feature, this.index);
```

Comment:
@jfirebaugh Is it possible to point out where the insertion into feature index happens for symbols in SymbolBucket and/or how do geometries for symbol features make into the index used for queryRenderedFeatures? 

The reason I am asking you this is because in an older version of GL JS, this insertion used to happen in WorkerTile and it was explicitly disabled for symbols. The only way to work around it and get symbols returned by queryRenderedFeatures was to comment out this condition.